packages/drivers/routerlicious-driver/src/test/documentService.spec.ts

```diff
@@ -142,4 +162,42 @@ describe("DocumentService", () => {
 		assert.equal(documentService.policies?.summarizeProtocolTree, true);
 		stubbedDeltaConnectionCreate.restore();
 	});
+
+	it("DocumentDeltaStorageService sends fetchReason along with fetchMessages", async () => {
+		// Create fake restWrapperBase
+		const restWrapperBase = new MockRestWrapper();
+		stub(restWrapperBase, "get").callsFake(async (url, headers) => {
```

Comment:
It's interesting that you created a MockRestWrapper (not important, it's more of a fake than mock), and you then stub out a response.

For your learning, it may be beneficial to figure out a way to reduce the code here.